packages/account/src/Components/trading-assessment/trading-assessment-form.jsx

```diff
@@ -220,7 +224,10 @@ const TradingAssessmentForm = ({
                                             {should_display_previous_button && (
                                                 <Button
                                                     has_effect
-                                                    onClick={displayPreviousPage}
+                                                    onClick={() => {
+                                                        setErrors({});
```

Comment:
Why are we setting the error object to `{}` when we move to previous?

Reply:
Because the form itself contains all fields from all steps during assessment. when we are getting the error for current field the error object is not empty. so once we are navigating to previous page, formik should not have errors. then when we click to next page again we will pass validation again